$NetBSD: patch-an,v 1.1 2007/03/18 08:38:14 rillig Exp $

Don't create an endless loop when the configure script failed. Since
bmake doesn't know about tail recursion, this is a slow fork bomb.

--- Makefile.orig	2005-10-07 18:51:32.000000000 +0200
+++ Makefile	2007-03-18 09:36:06.000000000 +0100
@@ -49,7 +49,7 @@ SUBDIRS = include libxfs libxlog libxcmd
 
 default: $(CONFIGURE)
 ifeq ($(HAVE_BUILDDEFS), no)
-	$(MAKE) -C . $@
+	exit 1
 else
 	$(SUBDIRS_MAKERULE)
 endif
